src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22032.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue22032 : _IssuesUITest
+	{
+		public Issue22032(TestDevice device) : base(device) { }
+
+		public override string Issue => "Shell FlyoutItem Tab Selected Icon Color not changing if using Font icons";
+
+		[Test]
+		[Category(UITestCategories.TabbedPage)]
+		public async Task SelectedTabIconShouldChangeColor()
+		{
+			App.WaitForElement("button");
+
+			App.Click("button");
+
+			await Task.Delay(500);
```

Comment:
instead of a delay can we `WaitForElement` on a control inside the switched tab?

Reply:
Sure!